src/elementPlugins/SummaryViewWrapper.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { ErrorBoundary, pluginProvider } from '.'
+import { Header } from 'semantic-ui-react'
+import { SummaryViewWrapperProps, PluginComponents } from './types'
+
+const SummaryViewWrapper: React.FC<SummaryViewWrapperProps> = (props) => {
+  const { element, value } = props
+  const {
+    parameters,
+    category,
+    code,
+    elementTypePluginCode: pluginCode,
+    isEditable,
+    isRequired,
+    isVisible,
+  } = element
+
+  if (!pluginCode || !isVisible || category === 'INFORMATION') return null
+
+  const { SummaryView }: PluginComponents = pluginProvider.getPluginElement(pluginCode)
+
+  // TO-DO: Provide a Default SummaryView (just label/text) if SummaryView not provided in plugin
```

Comment:
Maybe a good idea to already create an issue to do that? Also, please when creating new issues use the label to link with the epic - so it's easier to track them to be listed inside the epic, so we know how much we have done in each area.

Reply:
Yup, issue exists -- #129. Have just added Epic tag.

Reply:
Thanks!